pages/AddEditCard.js

```diff
@@ -0,0 +1,324 @@
+  const [actualAmount, setActualAmount] = useState(0);
+  const [maxAmount, setMaxAmount] = useState(0);
+
+  const decrementMinAmount = () => {
+    if (minAmount > 0) setMinAmount(minAmount - 1);
+  };
+
+  const incrementMinAmount = () => {
+    setMinAmount(minAmount + 1);
+  };
+
+  const decrementActualAmount = () => {
+    if (actualAmount > 0) setActualAmount(actualAmount - 1);
+  };
+
+  const incrementActualAmount = () => {
+    setActualAmount(actualAmount + 1);
+  };
+
+  const decrementMaxAmount = () => {
+    if (maxAmount > 0) setMaxAmount(maxAmount - 1);
+  };
+
+  const incrementMaxAmount = () => {
+    setMaxAmount(maxAmount + 1);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const product = {
+      productName,
+      unit,
+      category,
+      minAmount,
+      actualAmount,
+      maxAmount,
+    };
+    console.log(product);
+  };
+  return (
+    <AddCardStyled>
+      <FormStyled onSubmit={handleSubmit}>
+        <LabelStyled>Neues Produkt</LabelStyled>
+        <p>Produktname</p>
+        <StyledInput
+          type="text"
+          required
+          name="productName"
+          placeholder="Produktname"
+          value={productName}
+          onChange={(event) => setProductName(event.target.value)}
+        />
+        <p>Maßeinheit</p>
+        <StyledSelect
+          required
+          placeholder="Maß z.B. kg, Pkg, Flasche, Stk"
+          value={unit}
+          onChange={(event) => setUnit(event.target.value)}
+        >
+          <StyledOption defaultValue="selected">
+            Wähle eine Einheit
+          </StyledOption>
+          <StyledOption>
+            {units.map((unit) => {
+              return unit.name;
+            })}
+          </StyledOption>
+        </StyledSelect>
+        <p>Kategorie</p>
+        <StyledSelect
+          required
+          value={category}
+          onChange={(event) => setCategory(event.target.value)}
+        >
+          <StyledOption defaultValue="selected">
+            Wähle eine Kategorie
+          </StyledOption>
+          <StyledOption>
+            {categories.map((category) => {
+              return category.name;
+            })}
+          </StyledOption>
+          {/* <StyledOption value={categories[1].name}>
+            {categories[1].name}
+          </StyledOption>
+          <StyledOption value={categories[2].name}>
+            {categories[2].name}
+          </StyledOption>
+          <StyledOption value={categories[3].name}>
+            {categories[3].name}
+          </StyledOption>
+          <StyledOption value={categories[4].name}>
+            {categories[4].name}
+          </StyledOption>
+          <StyledOption value={categories[5].name}>
+            {categories[5].name}
+          </StyledOption> */}
+        </StyledSelect>
+        <p>Mindesbestand</p>
+        <AmountStyle>
+          <DecrementButton
+            onClick={(event) => decrementMinAmount(event.target.value)}
```

Comment:
`decrementMinAmount` does not take an argument
```suggestion
            onClick={(event) => decrementMinAmount()}
```